Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN rustup self update
 RUN cargo build --all --release
 
 # Create a new container from scratch to reduce image size
-FROM debian:buster-slim
+FROM debian:bullseye
```

Comment:
Had to change update this due to https://github.com/rust-lang/docker-rust/issues/77